dnx-on-ubuntu/install-dnx.sh

```diff
@@ -0,0 +1,56 @@
+admin_username=$1
```

Comment:
For templates with scripts, see organization recommendation https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/images/namingConvention.png